### ISSUE TITLE
Optimize entries in the manifest.json

### ISSUE THREAD

**Issue by riw** [JonasKruckenberg/imagetools#752]
Hi

first of all - thanks for your work! I want to suggest to add the originalFilename parameter when emitting the asset file (ideally including the path and query string - like the module id) so that the manifest.json file is more correct.

**Comment by benmccann** [comment]
I don't think we touch the manifest.json directly, so not sure how to affect it. PRs welcome
